sandy_sort_lists.py

```diff
@@ -0,0 +1,94 @@
+import config
+import requests
+import json
+
+
+COLOR_KEYS = {
+	"green": 1,
+	"yellow": 2,
+	"orange": 3,
+	"red": 4,
+	"purple": 5,
+	"blue": 6,
+	"sky": 7,
+	"lime": 8,
+	"pink": 9,
+	"black": 10
+}
+BOARD_IDS_URL = "https://api.trello.com/1/members/{username}/boards"
+GET_LISTS_URL = "https://api.trello.com/1/boards/{board_id}/lists"
+GET_CARDS_URL = "https://api.trello.com/1/lists/{list_id}/cards"
+PUT_CARDS_URL = "https://api.trello.com/1/cards/{card_id}/pos"
+BASIC_PARAMS = {
+        "key": config.TRELLO_APP_KEY,
+        "token": config.TRELLO_APP_TOKEN
+        }
+
+
+def get_key_value(card):
+	try:
+		color = COLOR_KEYS[card["labels"][0]["color"]]
+	except IndexError, e:
+		color = 0
+	return color
+
+
+def get_lists_from_board(board_id):
+	resp = requests.get(GET_LISTS_URL.format(board_id=board_id), params=BASIC_PARAMS)
+	json_resp = json.loads(resp.text)
+
+	lists = []
+	for l in json_resp:
+		lists.append(l["id"])
+
+	return lists
+
+
+def sort_list(list_id):
+	resp = requests.get(GET_CARDS_URL.format(list_id=list_id), params=BASIC_PARAMS)
+	json_resp = json.loads(resp.text)
```

Comment:
I think it's more helpful and clear to name what this return object is, rather than `json_resp`. Like maybe `card_dicts` or something